README.md

```
@@ -119,6 +121,23 @@ such on a terminal: emphasis, headings, lists, code and links the terminal can
 open. Piped or redirected the description is the source text exactly as it was
 written, and so it is under `--json` and `--compact`.
 
+### Comments and activity
+
+Each issue has an append-only timeline containing Markdown comments and compact
+system records of its changes.
+
+```console
+$ awb comment add awb-5c1d84 --body "Reproduced with an empty token stream."
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)